source/crud/crud.rst

```diff
@@ -445,7 +445,7 @@ Read
     maxAwaitTimeMS: Optional<Int64>;
 
     /**
-     * Maximum number of documents or index keys to scan when executing the query.
+     * Deprecated : 4.0. Maximum number of documents or index keys to scan when executing the query.
```

Comment:
I'll make one final appeal for `@deprecated 4.0` instead of changing description paragraphs. I think it stands out more when scanning the spec and it makes for a more concise diff (adding a deprecation line without modifying existing content).

Reply:
Agreed.  I don't like it where it is either. How about:

    @deprecated 4.0
    @removed 4.2

Reply:
I like that a lot. And if we need to track when options are introduced, `@since 3.6` also works. That might clean up some of the existing prose if anyone was inclined to go back and refactor :)

Reply:
Updated.